src/update-manager.js

```diff
@@ -1,4 +1,4 @@
-// Joe Presbrey <presbrey@mit.edu>
+UpdateManager// Joe Presbrey <presbrey@mit.edu>
```

Comment:
Surely a mistake?

Reply:
No, Joe wrote the original module -- I guess we could make it clearer

Reply:
Sure, but `UpdateManager` is not commented, it's a JavaScript statement. It's harmless because it just evaluates to the class below, but it doesn't do anything so should be removed.

Reply:
Oh yes that was a typo from missig the Command/F  I suspect ..